docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is enabled by default for Teleport Team users.
+You can skip the setup process here and start using Access Graph right away
+by signing up for a [Teleport Team trial](https://goteleport.com/signup/).
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v14.3.4 or later.
+- An updated `license.pem` with Teleport Policy enabled.
+- Docker version v(=docker.version=) or later.
+- A PostgreSQL database server v14 or later.
+  - Access Graph needs a dedicated [database](https://www.postgresql.org/docs/current/sql-createdatabase.html) to store its data.
+    The user that TAG connects to the database with needs to be an owner of this database, or have similar broad permissions:
+    at least the `CREATE TABLE` privilege on the `public` schema, and the `CREATE SCHEMA` privilege.
+  - Amazon RDS for PostgreSQL is supported.
+- A TLS certificate for the Access Graph service
+  - The TLS certificate must be issued for "server authentication" key usage,
+    and must list the IP or DNS name of the TAG service in an X.509 v3 `subjectAltName` extension.
+- The node running the Access Graph service must be reachable from Teleport Auth and Proxy services.
+
+</TabItem>
+</Tabs>
+
+## Step 1/3. Set up the Teleport Access Graph service
+
+You'll need a copy of your Teleport cluster's host certificate authority (CA) on the machine that hosts the Access Graph service.
+TAG requires incoming connections to be authenticated via host certificates that the host CA issues to Auth and Proxy services.
+
+The host CA can be retrieved and saved into a file in one of the following ways:
+
+<Tabs>
+<TabItem label="Via curl">
+```console
+$ mkdir /etc/access_graph
+$ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=tls-host' > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+
+<TabItem label="Via tctl">
+```console
+$ mkdir /etc/access_graph
+$ tsh login --proxy=<Var name="teleport.example.com" />
+$ tctl get cert_authorities --format=json \
+    | jq -r '.[] | select(.spec.type == "host") | .spec.active_keys.tls[].cert' \
+    | base64 -d > /etc/access_graph/teleport_host_ca.pem
+```
+</TabItem>
+</Tabs>
+
+Then, create a configuration file for the TAG service, similar to this:
```

Comment:
We should make it more explicit that this configuration file should be on the server running the TAG service

Reply:
We do mention it at the start of this section https://github.com/gravitational/teleport/pull/37661/files/e9b522fce41ba99443d5151c2e8ff9482b124336#diff-351d93272d6e47e36224548eec736412a333de6d70c937e1512ccfaf61faa8f0R35 , but I'll re-iterate here.